build.sbt

```diff
@@ -1,7 +1,7 @@
 inThisBuild(
   Seq(
-    scalaVersion := "2.12.11",
-    crossScalaVersions := Seq("2.12.11", "2.13.1"),
+    scalaVersion := "2.13.2",
+    crossScalaVersions := Seq("2.12.11", "2.13.2"),
```

Comment:
Need to include `2.12.11` in `crossScalaVersions` if you change the main scala version.

Reply:
Erhm, early morning review, never mind.